overthebox/files/bin/otb-action-configure

```diff
@@ -98,6 +98,30 @@ case "$tun_app" in
 		;;
 esac | uci -q batch
 
+delete_tracking_conf() {
+	generator=
+	config_get generator "$1" generated_by
+	[ "$generator" = "$ME" ] && uci -q delete "otb-tracker.$1"
+}
+
+# shellcheck disable=SC1091
+. /lib/functions.sh
+config_load otb-tracker
+config_foreach delete_tracking_conf track
+case "$tun_app" in
+	glorytun|glorytun_hybrid)
+		tracked_interfaces=$(_get glorytun_conf | jq -r '@text "\(.dev) \(.additional_interfaces[].dev)"');;
+	glorytun_mud)
+		tracked_interfaces=$(_get glorytun_mud_conf | jq -r '@text "\(.dev)"');;
+esac
+for tracked_interface in $tracked_interfaces; do
+	uci -q batch <<-EOF
+	set otb-tracker.$tracked_interface=track
+	set otb-tracker.$tracked_interface.tries=5
+	set otb-tracker.$tracked_interface.generatedby=$ME
```

Comment:
Anyway, line 24 of the file, all the config generatedby "$ME" is already deleted, including your tracker config